public/js/util.js

```diff
@@ -0,0 +1,247 @@
+var firstFloor = document.getElementById("floor1");
+var secondFloor = document.getElementById("floor2");
+var modal = document.querySelector('.modal');
+var modalOverlay = document.querySelector('.modal-overlay');
+
+var events;
+
+// An AJAX request to get the data from the event-data.json file and parsing it
+$.getJSON({
+    url: "./event-data.json",
+    async: false
+}, function(data) {
+    events = data;
+});
+
+// Transforming the start and end times into date objects
+for (event of events) {
```

Comment:
This is going to have a race condition that is definitely gonna fail – should put in callback after getting JSON